tests/js/client/server_parameters/log-json-format-info-cluster.js

```diff
@@ -0,0 +1,125 @@
+/*jshint globalstrict:false, strict:false */
+/* global getOptions, assertTrue, arango, assertMatch, assertEqual, instanceManager */
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test for server startup options
+/// DISCLAIMER
+///
+/// Copyright 2014-2022 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Julia Puget
+////////////////////////////////////////////////////////////////////////////////
+
+const fs = require('fs');
+const db = require('internal').db;
+const getCoordinatorEndpoints = require('@arangodb/test-helper').getCoordinatorEndpoints;
+const cn = "testCollection";
+const dbName = "maçã";
+
+if (getOptions === true) {
+  return {
+    'log.use-json-format': 'true',
+    'log.output': 'file://' + fs.getTempFile() + '.$PID',
+    'log.prefix': 'PREFIX',
+    'log.thread': 'true',
+    'log.thread-name': 'true',
+    'log.role': 'true',
+    'log.line-number': 'true',
+    'log.ids': 'true',
+    'log.hostname': 'HOSTNAME',
+    'log.process': 'true',
+    'log.in-memory': 'true',
+    'log.level': 'info',
+    'log.escape-unicode-chars': 'true',
+    'database.extended-names-databases': 'true',
+    'server.authentication': 'false',
+  };
+}
+
+
+const jsunity = require('jsunity');
+
+function LoggerSuite() {
+  'use strict';
+
+  return {
+
+    setUp: function() {
+      db._useDatabase("_system");
+      db._createDatabase(dbName);
+      db._useDatabase(dbName);
+      const c = db._create(cn, {numberOfShards: 3});
+      let props = c.properties();
+      assertEqual(3, props.numberOfShards);
+    },
+
+    tearDown: function() {
+      db._useDatabase("_system");
+      db._dropDatabase(dbName);
+    },
+
+    testLogClusterInfoInFile: function() {
+      let coordinatorEndpoint = getCoordinatorEndpoints()[0];
+      arango.reconnect(coordinatorEndpoint, db._name(), "root", "");
```

Comment:
this "reconnect" will potentially have a side effect that at before this test we may be connected to a different server than after this test.
If a test reconnects, it should make sure in its `tearDown` / `tearDownAll` that the pre-test state is restored when the test completes.

Reply:
Addressed